### PR TITLE
docs: add readme

### PR DESCRIPTION
**Pull request by SrIzan10**
add readme (you can merge)

### CODE REVIEW

**Comment by SrIzan10** [comment]
oops sorry for not using the right conventional commit message at first 😩

**Comment by SrIzan10** [comment]
okay it's ready

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]
Add some awesome badges and mention that this is maintained by the _awesome_ community. and change all secondary headers from `###` to `##`

**Comment by SrIzan10** [comment]
will do later today

**Comment by SrIzan10** [comment]
I didn't have time yesterday, let me finish it.

https://canary.discord.com/channels/889026545715400705/980390255574282282/1019178307155152999

also read that

**Comment by SrIzan10** [comment]
kay done, ready to merge

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]
See [this](https://github.com/sindresorhus/awesome/blob/main/awesome.md) as well, tyvm

**Comment by SrIzan10** [comment]
so we're applying for awesome right?

**Comment by SrIzan10** [comment]
sorry for not fixing `###` before. I'm using the Github Web Editor, got timed out while commiting and then I lost the work.
should be fixed with the last commit.

**Comment by SrIzan10** [comment]
@jacoobes please review?

**Comment by SrIzan10** [comment]
also, please enable server widgets so the discord badge can work

**Review by EvolutionX-10 — COMMENTED** [review]
Looks fine to me, server widget is enabled now, there are few things in my mind but I'll add them later. Let seren review it

**Comment by SrIzan10** [comment]
cool, thank you very much evo!
image

**Comment by SrIzan10** [comment]
also, can you approve it? seems like the requested changes are still there!

**Review by EvolutionX-10 — DISMISSED** [review]

**Comment by SrIzan10** [comment]
cool

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+As easy as running `sern plugins` with the [CLI](https://github.com/sern-handler/cli).
+
+You can also install them by downloading the file into your project.
+
+Then, read the comments of the plugin file to know how to use them or run `/plugin` on the discord server
+
+## Contributing
+
+Create your plugin by reading the docs, then open a PR.
+Code it in typescript, and a GitHub Action will convert the plugin to JavaScript automatically when merged.
```
Comment (jacoobes) [inline README.md]: TypeScript, not typescript

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
-## TODO
+[![Awesome](https://awesome.re/badge-flat.svg)](https://awesome.re) ![discord](https://img.shields.io/discord/889026545715400705) ![reposize](https://img.shields.io/github/languages/code-size/sern-handler/awesome-plugins?label=repo%20size) ![forks](https://img.shields.io/github/forks/sern-handler/awesome-plugins?style=social) ![stars](https://img.shields.io/github/stars/sern-handler/awesome-plugins?style=social)
+# sern Plugins
+
+An awesome list of plugins made by the awesome community, for the awesome community who use the sern framework to develop their Discord Bots.
+
+## How to install
+
```
Comment (jacoobes) [inline README.md:7]: Make a warning saying you must include sern.config.json!, if you are installing via cli

**Review by jacoobes — CHANGES_REQUESTED** [review]

**Comment by SrIzan10** [comment]
will do

File: README.md (hunk 1 above)
Comment (SrIzan10) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (SrIzan10) [reply README.md:7]: done

**Comment by jacoobes** [comment]
cheers

**Review by jacoobes — APPROVED** [review]

**Comment by SrIzan10** [comment]
lets hope evo approves

**Review by EvolutionX-10 — APPROVED** [review]

**Comment by SrIzan10** [comment]
🎉 thanks!

**Pull request merged**